css/style.css

```diff
@@ -1,4 +1,224 @@
+  cursor: pointer;
+  color: #E51937;
+  box-shadow: 0 2px 0 #E51937;
+  opacity: .75;
+  transition: 0.4s;
+}
+
+.menubar__link_red:nth-child(1) {
+margin-left: 258.14px;
+}
+
+.menubar__link:nth-child(2) {
+  margin-left: 63.14px;
+}
+
+.menubar__link:nth-child(3){
+  margin-left: 24.86px;
+}
+
+.menubar__link:nth-child(4) {
+  margin-left: 44px;
+}
+
+.menubar__link:nth-child(5) {
+  margin-left: 46px;
+}
+
+.menubar__link:nth-child(6) {
+  margin-left: 221.14px;
+}
+
+.menubar__link:nth-child(7) {
+  margin-left: 30px;
+}
+
+.button:nth-child(8) {
+  margin-left: 30px;
+}
+
+.button {
+  display: block;
+  background-color: #ffffff;
+  border: non;
+  width: 24px;
+  height: 15px;
+  position: relative;
+}
+
+.button .button__line, 
+.button .button__line:before,
+.button .button__line:after {
+  position: absolute;
+  display: block;
+  top: 50%; margin-top: -1px;
+  left: 50%; margin-left: -12px;
+  width: inherit;
+  height: 2px;
+  background-color: #E51937;
+}
+
+.button__line:before, .button__line:after {
+  content: '';
+}
+
+.button__line:before {
+ transform: translateY(-6px);
+}
+
+.button__line:after {
+  transform: translateY(6px);
+ }
+
+.button:hover {
+  cursor: pointer;
+  box-shadow: 0 0 5px #473c3c;
+  border: 1px solid #E51937;
+  transition: 0.5s;
+}
+.content {
+  position: relative;
+  /* background: url("../img/jpg/Tesla\ logo\ car.jpg"); */
+}
+
+.content__main-title, .content__title {
+  font-family: Montserrat;
+  font-style: normal;
+  text-align: center;
+}
+
+.content__main-title {
+  position: absolute;
+  font-weight: 600;
+  font-size: 90px;
+  margin: auto;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  padding-top: 60px;
+  z-index: 2;
+}
+
+.content__title {
+  position: absolute;
+  font-weight: 500;
+  font-size: 15px;
+  margin: auto;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  padding-top: 150px;
+  z-index: 2;
+}
+
+.content__back {
+  transform: translateY(40px);
+}
+
+.content__data-car {
+ display: flex;
+ padding-top: 60px;
+ justify-content: space-between;
+ align-items: center;
+ margin-left: 321px;
+}
+
+.data__speed-block:before, .data__road-block:before {
+  content: '';
+  display: inline-block;
+}
+
+.data__speed-block:before {
+  background-image: url('/img/png/Speed.png');
+  background-size: 46px 34.13px;
+  width: 46px;
+  height: 34.13px;
+  background-repeat: no-repeat;
+  margin-top: px;
+}
+
+.data__road-block:before {
+  background-image: url('/img/png/Road\ icon.png');
+  background-size: 30.5px 50px;
+  width: 30.5px;
+  height: 50px;
+  background-repeat: no-repeat;
+}
+
+.data__figures {
+  font-weight: 600;
+  font-size: 35px;
+}
+
+.data__speed {
+  font-weight: 600;
+  font-size: 20px;
+}
+
+.data__specifications {
+  display: inline-block;
+  font-size: 12px;
+}
+
+.testdrive__img {
+  width: 118px;
+  height: 22px;
+  margin-left: 105px;
+}
+
+.order-btn {
+  margin: 0 321px 0 40px;
+  border: 2px solid #E51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)